issue2/html_task.js

```diff
@@ -0,0 +1,38 @@
+var webdriver = require('selenium-webdriver');
+var assert = require('chai').assert;
+var { Builder, By } = require("selenium-webdriver");
+
+describe("Testing Scholars Project", function () {
+    this.timeout(5 * 1000 * 60);
+    var driver = new webdriver.Builder()
+        .forBrowser('chrome')
+        .build();
+
+    before(function setupWebdriver(done) {
+        driver.get("file:///C:/Users/oemma/mocha_test/Dufuna-Fem/submissions/desolaOluseye/HTML/productPage.html").then(done)
+    });
+
+    it("test case: check if Price is present", async () => {
+        const element = await driver.findElement(By.css("h3"));
```

Comment:
Are we sure that the price element will always be enclosed in an `h3` tag? Was the given task this specific?